src/main/java/com/endava/soa/pageObject/AdaugaAnuntPage.java

```diff
@@ -0,0 +1,9 @@
+package com.endava.soa.pageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AdaugaAnuntPage {
+    @FindBy(xpath = "//a[contains(text(),'Aparate telefonice')]")
+    private WebElement aparateTelefonice;
```

Comment:
What will happen when we will use Russian language?
I suggest to find by css class or id

Reply:
outdated